Dockerfile

```diff
@@ -15,4 +15,6 @@ EXPOSE 3000
 
 COPY . /code
 
+RUN [ "npm", "run", "build-css" ]
```

Comment:
Could/should this be done as part of `npm start` i.e. change `package.json` to be `"start": "npm run build-css && node app.js"`